reference/upgrades.md

```diff
@@ -0,0 +1,5 @@
+## Upgrading Mayastor
+
+Mayastor v2.0.0 does not support upgrades from previous versions. The installation of the new version should be preceded by the removal of any earlier releases.
```

Comment:
upgrade instead of upgrades ?
"removal of any earlier version" ?

Reply:
These changes have been incorporated in https://github.com/openebs/mayastor-docs/pull/109